automated_evaluation/scripts/build_competitor_code.py

```diff
@@ -25,32 +25,32 @@ def main():
             data = yaml.safe_load(stream)
         except yaml.YAMLError:
             print("Unable to parse yaml file")
-            exit()
+            sys.exit()
 
     # Store data from yaml filyaml_path
     try:
         repository = data["github"]["repository"]
     except KeyError:
         print("Unable to find repository link")
-        exit()
+        sys.exit()
     
     try:
         token = data["github"]["personal_access_token"]
     except KeyError:
         print("Unable to find personal_access_token")
-        exit()
+        sys.exit()
 
     try:
         package_name = data["competition"]["package_name"]
     except KeyError:
         print("Unable to find package_name")
-        exit()
+        sys.exit()
 
     try:
         launch_file = data["competition"]["launch_file"]
     except KeyError:
         print("Unable to find launch_file")
-        exit()
+        sys.exit()
     
     # Clone the repository
     clone_cmd = f"git clone https://{token}@{repository} ~/ariac_ws/src/{package_name}"
```

Comment:
It is convenient (in terms of brevity) if cloning target path is set using team_name instead package_name. (Hopefully assume that it doesn't have side effects on other scripts or would need rectifying those places as well.)

Reply:
Why is it more convenient to use team_name?

Reply:
Nothing serious just personal preference. We favor following code structure since we have multiple packages in our solution (Some being pulled from 3rd party repos to subdirectories as well.)
```
ariac_ws
|-src
  |-ariac (main ariac repo)
  | ...
  |-team_name (team repo)
    |-package_name (main package where launch file resides)
    |  |-CMakeLists.txt
    |    ...
    |-some_other_package
    |  |-CMakeLists.txt
    |    ...
    |-directory_containing_more_packages
      |-more_package_name
      |  |-CMakeLists.txt
      |    ...
      |-some_more_other_package
      |  |-CMakeLists.txt
          ...
```
But above makes it like this
```
ariac_ws
|-src
  |-ariac (main ariac repo)
  | ...
  |-package_name (team repo)
    |-package_name  (main package where launch file resides)
    |  |-CMakeLists.txt
    |    ...
    |-some_other_package
    |  |-CMakeLists.txt
    |    ...
    |-directory_containing_more_packages
      |-more_package_name
      |  |-CMakeLists.txt
      |    ...
      |-some_more_other_package
      |  |-CMakeLists.txt
          ...
```
As I said it's just cosmetic, feel free to ignore the request.

Reply:
I see now, yes I agree this makes more sense for teams with multiple packages. I will update this.